src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -77,9 +77,11 @@ services:
         tags:
             - controller.service_arguments
 
-    Ibexa\Bundle\AdminUi\Controller\ContentViewController: ~
+    Ibexa\Bundle\AdminUi\Controller\ContentViewController:
+        parent: Ibexa\Contracts\AdminUi\Controller\Controller
```

Comment:
I don't really immediately see how setting this as parent definition helps. Especially since this definition lies in a completely different file it seems.

Have you considered using [`_instanceof`](https://symfony.com/doc/current/service_container/tags.html#autoconfiguring-tags) to prevent forgetting that this `parent` has to be added to each controller of this type?

Reply:
Actually I haven't. Til this point we have followed adding parent controller to all of the admin-ui-related ones. I only aimed to fix the ones that slipped between cracks and which I tracked down while working on enabling new security layer. 

I would say that if this approach needs revisiting we should work on that separately as it's a bit out of scope of security-related changes.

Unless you had something else in mind.